### DISCUSSION TITLE
Help with checking content on a separate tab

### DISCUSSION THREAD

**Discussion by JL082** [cypress-io/cypress#21105] [General]
Good Afternoon,

I'm attempting to validate on a page that happens to open in a new tab, when I do so it opens up in a separate window that Cypress cannot test against. Would anyone know how I can have cypress validate this new tab or a way to call this URL in a separate test. If I close the main window I'm unable to go to the secondary window that pops up.

Below is the isolated link that I'm attempting to open that redirects to a separate browser tab.
				<a href="javascript:openMemberOrHradmin('/member/control/gateway?personHandle=31936193787');">Open in Member Role</a>

**Comment by JL082** [comment]
![anchor](https://user-images.githubusercontent.com/103780768/163629423-801467a6-959a-43ce-bcd8-4a0696a899fe.PNG)

**Comment by geromegrignon** [comment]
Hi @JL082,

As explained [here](https://docs.cypress.io/guides/references/trade-offs#Multiple-tabs), there is no multi-tabs support.
